src/main/java/models/Bid.java

```diff
@@ -45,4 +45,9 @@ public String toString() {
                 ", id=" + id +
                 '}';
     }
+
+    @Override
+    public int compareTo(Bid bid) {
```

Comment:
I think a comparison through `id` would be more appropriate.

Reply:
Well, the idea was to compare Bid amounts, so...